UX-Guide-Metadata/draft/principles/index.html

```diff
@@ -280,11 +280,34 @@ <h4>Metadata techniques</h4>
 			<section id="conformance-group">
 				<h3>Conformance</h3>
 				
-				<p>TBD</p>
-				
-				<p>TBD</p>
-				<h4>Examples</h4>
-                <p>TBD</p>
+<p>Description: The conformance group provides the assertions made concerning the accessibility of the publication. At a minimum, the specification which it conforms to and the party doing the certification must be provided. It is also possible to determine if the certification is done by a third party. If the dc:publisher and a11y:certifiedBy property are different, then  this indicates the certification was done by a third party. The credentials of the certifying party and the  date of the evaluation will be of interest and should be provided. If there is a link to a report of the evaluation, that can also be provided. </p>
+ 
+<dl>
+<dt>conformsTo</dt><dd> REQUIRED - Identifies the accessibility specification and the conformance level to which  the publication  assertians are made. When the publication conforms to more than one specification, additional conformsTo statements may be provided. The EPUB Accessibility specification 1.1 uses natural language descriptions in the conformsTo statement, which means these values can be provided directly, and no translation will be necessary.</dd> 
+<dt>certifiedBy</dt>  <dd>REQUIRED -  This provides information about the organization providing the certification review or process for certification.</dd>
+<dt>Third party certification</dt><dd>This statement can be determined by comparing the values of the dc:publisher and a11y:certifiedBy properties. If the values are different, then third party certifications are true.</dd> 
+<dt>certifierCredential </dt><dd>If the certifier has a badge or a credential, then the value is provided.</dd>
+<dt>Certification date</dt><dd>If the date of the certifiers   evaluation is provided, then this would be of interest. This is associated with the certifier through the refines value, e.g., refines="#certifier" followed by the date.</dd>
```

Comment:
```suggestion
<dt>Certification date</dt><dd>If the date of the certifier's evaluation is provided, then this would be of interest. This is associated with the certifier through the refines value, e.g., refines="#certifier" followed by the date.</dd>
```